python/Ganga/new_tests/GPI/Bugs/TestSavannah1006.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from ..GangaUnitTest import GangaUnitTest
+
+class TestSavannah10006(GangaUnitTest):
```

Comment:
The filename should be `TestSavannah10006.py` (three `0`s) and the old test should be removed at the same time.